packages/@okta/vuepress-site/code/rest/index.md

```diff
@@ -1,94 +1,100 @@
 meta:
   - name: description
-    content: Find documentation to get started with Okta REST APIs and learn how to import a collection and send requests in Postman.
+    content: Get started with Okta REST APIs and learn how to import a collection and send requests in Postman.
 ---
 
-A great way to learn an API is to issue requests and inspect the responses. You can easily use our Postman collections to do just that.
+A great way to learn an API is to issue requests and inspect the responses. You can easily use our Postman collections to do just that. To use these collections, you need to set up your local environment and import a collection. Then, you can send a test request and verify the results.
 
-![Postman and an Okta Collection](/img/okta_postman_logo.png "Postman and an Okta Collection")
+## Sign up for Okta
 
-To use these collections, you'll need to set up your local environment and import the collections. Then, you can send a test request and verify the results.
+You need a free Okta developer edition org to get started. Don't have one? [Create an org for free](https://developer.okta.com/signup). When you create a new Okta org, the org is assigned a base URL such as `dev-12345.okta.com`. This is your unique subdomain in Okta.
 
-## Sign Up for Okta
-
-You'll need a free Okta developer organization to get started. If you don't have one already, [sign up](https://developer.okta.com/signup/) to create one. When you create a new Okta organization, it will be assigned a base URL like `dev-12345.okta.com`. This is your unique subdomain in Okta.
-
-## Set Up Your Environment
+## Set up your environment
 
 1. [Create an API token](/docs/guides/create-an-api-token/) for your org.
 
-  > **Note:** Alternatively, you can [create an OAuth 2.0 access token](/docs/guides/implement-oauth-for-okta/) for use with a number of Okta endpoints. This is an EA feature that allows you to interact with Okta APIs using scoped OAuth 2.0 access tokens. Each access token enables the bearer to perform specific actions on specific Okta endpoints, with that ability controlled by which scopes the access token contains.
+> **Note:** Alternatively, you can [create an OAuth 2.0 access token](/docs/guides/implement-oauth-for-okta/) for use with a number of Okta endpoints. This is a feature that allows you to interact with Okta APIs using scoped OAuth 2.0 access tokens. Each access token enables the bearer to perform specific actions on specific Okta endpoints, with that ability controlled by which scopes the access token contains.
 
-1. [Install the Postman app](https://www.getpostman.com/apps).
-1. Launch Postman and click the **Import** button. Select **Import From Link**, and paste this link into the textbox: `https://developer.okta.com/docs/api/postman/example.oktapreview.com.environment`
-![Importing the Okta Example Environment](/img/import_enviro.png "Importing the Okta Example Environment")
+2. [Install the Postman app](https://www.getpostman.com/apps).
```

Comment:
If you use just "1." as the marker, you won't have to renumber the steps if something changes in the future. Markdown will automatically number them in sequential order.

Reply:
@IanHakes-okta that doesn't seem to be working right now, so I am numbering. If you look at the current guide - the numbering is broken: https://developer.okta.com/code/rest/